docs/ui-extensions-sdk-frontend.md

```diff
@@ -270,3 +271,58 @@ Listens for DOM changes and calls `updateHeight()` when the size changes.
 
 ##### `window.stopAutoResizer()`
 Stops resizing the iframe automatically.
+
+
+## `extension.dialogs`
+This object provides methods for opening UI dialogs:
+
+##### `dialogs.selectSingleEntry(options)`
+Opens a dialog for selecting a single entry. It returns a promise that is
+resolved with the selected entity or `null` if a user closes the dialog without
+selecting anything.
+
+`options` is an optional object configuring the dialog. Available `options`:
+
+* `locale` - code of a locale that you want to use. It'll be used to display
+  proper titles and descriptions. Defaults to the default space's locale
+* `contentTypes` - an array of content type IDs of entries that we want to
+  display in the selector. By default entries of all content types are displayed
+
+~~~js
+// display a dialog for selecting a single entry
+dialogs.selectSingleEntry().then((selectedEntry) => {})
+
+// select a single entry of "blogpost" content type
+// and display result in "de-DE" locale
+dialogs.selectSingleEntry({
+  locale: 'de-DE',
+  contentTypes: ['blogpost']
+}).then((selectedEntry) => {})
+~~~
```

Comment:
Please add _In upcoming release_. See the [developer docs](https://github.com/contentful/ui-extensions-sdk/blob/master/DEVELOPING.md) for more info